### PR TITLE
Refactor: renames to integrations and splits out issue integrations

### PR DESCRIPTION
**Pull request by axosoft-ramint**

### CODE REVIEW

File: src/plus/integrations/integration.ts [hunk 1]
```diff
@@ -74,15 +87,20 @@ export abstract class ProviderIntegration<
 		return this.id;
 	}
 
-	private get connectedKey(): `connected:${ProviderIntegration['key']}` {
+	private get connectedKey(): `connected:${HostingIntegration['key']}` {
```
Comment (d13) [inline src/plus/integrations/integration.ts:90]: Should this be `Integration['key']`?

Comment (axosoft-ramint) [reply src/plus/integrations/integration.ts]: Well spotted! Updated it 👍🏼

**Review by eamodio — APPROVED** [review]
:shipit:

**Review by KidiIT — COMMENTED** [review]
All reviewed and approved🫶❤️

**Pull request merged**